juniper_official/vpn/evpn/evpn-instance-check.rule

```diff
@@ -30,6 +30,8 @@
                 }
                 type string;
                 description "Mode of evpn Interface";
+                enumerate single-homed as 0;
```

Comment:
Is single homed a bad state?
If it is a good state it should be >=1

Reply:
single homed is the default value and is good state. updated with positive value